locale/en/locale.cfg

```diff
@@ -32,12 +32,11 @@ no-energy=Not enough energy to teleport.
 teleported=Teleported: __1__ to __2__ __3__
 teleport-failed=Unable to teleport: __1__ to __2__. [color=#ff0000]Suitable position not found[/color] __3__.
 
-[coe-launch-gui]
-button_show_statistics=Show Cities of Earth Statistics
-title_coe_ui=Cities of Earth Statistics
-button_close=Close
+[coe-stats-gui]
+button_statistics=Show Cities of Earth Statistics
+label_stats_title=Cities of Earth
 label_launches_title=Launches:
-label_launches_required=Required:
+label_launches_required=Required:  
```

Comment:
Extra space at the end here, This is an issue with IDE's that have nuke whitespace on :)

Reply:
I want that extra space so the numbers line up better.  Creating a table layout for two rows isn't worth it, IMO.

Reply:
I'll try the `__1__` mentioned below.